pkg/process/config/yaml_config.go

```diff
@@ -172,6 +172,21 @@ func (a *AgentConfig) loadSysProbeYamlConfig(path string) error {
 		a.Windows.DriverBufferSize = driverBufferSize
 	}
 
+	// If there is no network_config, assume it is an old version of the config, and enable
+	// the network check. This has the downside that if the network_config is deleted, network
+	// check will be enabled.  However, this seems like the best option for backwards compatibility.
+	//
+	// The network check won't be run if sysprobe isn't enabled so there's no need to check
+	// EnableSystemProbe here.
+	networkEnabled := true
+	if config.Datadog.IsSet(key(spNS, "network_config")) {
+		// System probe can be run without the network module as determined on the network_config.enabled value
+		networkEnabled = config.Datadog.GetBool(key(spNS, "network_config.enabled"))
+	}
+	if networkEnabled {
+		a.EnabledChecks = append(a.EnabledChecks, "Network")
+	}
+
```

Comment:
Can we log something here at info level that explains that the check is being implicitly enabled?